code/modules/cargo/packs/costumes_toys.dm

```diff
@@ -280,7 +280,7 @@
 
 /datum/supply_pack/costumes_toys/wizard
 	name = "Wizard Costume Crate"
-	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Nanotrasen would like to remind its employees that actually joining the Wizard Federation is subject to termination of job and life."
+	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Head Office would like to remind its employees that actually joining the Wizard Federation is subject to termination of job and life."
```

Comment:
Pretend to join the Wizard Federation with this full wizard outfit! Head Office would like to remind all employees that actually joining the Wizard Federation is subject to termination of job and life as per clause 47(c) of the ICC's magic and wizardry accord.